Marlin/Configuration.h

```diff
@@ -1767,6 +1767,14 @@
  */
 //#define SDSUPPORT
 
+/**
+ * USB HOST MSC
+ *
+ * USB Flash mounted in MCU USB OTG Host port
+ * you need uncomment both SDSUPPORT and following option at the same time or it won't work.
+ */
+//#define USB_HOST_MSC_FLASH_SUPPORT
```

Comment:
Rather than leave it up to the user to set this, it would be better to enable it automatically when it is required.